src/TextField/TextFieldLabel.js

```diff
@@ -4,15 +4,14 @@ import React, { PropTypes } from 'react';
 import { createStyleSheet } from 'jss-theme-reactor';
 import classNames from 'classnames';
 import { easing } from '../styles/transitions';
+import LabelBase from '../internal/LabelBase';
```

Comment:
I'm not sure to follow the reason behind this encapsulation. Isn't an early refactorisation?